docs/visual/e2e-testing/troubleshooting.md

```diff
@@ -95,3 +95,17 @@ capabilities = {
 ```
 
 Additional information on WebDriver timeouts can be found [here](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Errors/ScriptTimeout).
+
+### cropTo can not crop the target element
+
+#### element is not scrollable
+
+In this case, you could try to use the scrollAndStitchScreenshot approach. For example:
```

Comment:
If you're unable to crop the target element with `cropTo` and the element is not scrollable, you can use the `scrollAndStitchScreenshot` parameter. For example: